.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -36,6 +36,10 @@ What must a solution address in order to solve the problem? How do we know the s
   * **Will this change trigger SAW changes?** Changes to the state machine, the s2n_handshake_io code that controls state transitions, the DRBG, or the corking/uncorking logic could trigger SAW failures.
   * **Should this change be fuzz tested?** Will it handle untrusted input? Create a separate issue to track the fuzzing work.
 
+### Need By Date:
+
+Any information you can give us to help prioritize the issue is appreciated. However, we cannot guarantee that this issue will be fixed by your timeline.
+
```

Comment:
I think this misses calling out that we'd love a reason for the date. A date alone doesn't really help us with prioritization, but on the other hand someone opening an issue doesn't necessarily know what will help us with prioritization. Let's be clear about what we need.

My suggestion:
```suggestion
Do you have a date that you need this issue resolved by? What is the reason for that date, and what are the consequences of missing it? Any additional information you can provide to help prioritize the issue is appreciated. However, we cannot guarantee that this issue will be fixed by the requested date.

```